Library/Homebrew/cmd/update-report.rb

```diff
@@ -126,131 +127,137 @@ def output_update_report
 
       updated = true
 
-      old_tag = Settings.read "latesttag"
+      old_repository_version = Settings.read "latesttag"
```

Comment:
Please leave this name as-is. It's still a tag.

Reply:
I will remove commit